### DISCUSSION TITLE
Deamidation not supported?

### DISCUSSION THREAD

**Discussion by brvpuyve** [vdemichev/DiaNN#104] [Q&A]
Hi Vadim,

Sorry for having another question.... 

I am currrently working with the DPHL library (A DIA Pan-human Protein Mass Spectrometry Library for Robust Biomarker Discovery) in an .SPTXT format. Apparently, it contains deamidation as variable modification but it is not recognized by DIA-NN. I have tried to add it with the command --var UniMod: 7, 0.984016, but it does not work. Could you help me out with this issue?

Kind regards,
Bart 

![image](https://user-images.githubusercontent.com/35368580/117189243-406a8600-adde-11eb-8c4e-3a75fa1a8d6a.png)

**Comment by vdemichev** [comment]
Hi Bart,

It's recognised with any of these names:
"UniMod:7", "Deamidation (NQ)", "Deamidation", "de", "Dea", "+1", "+1.0"

If it's a different name in the library, need to let DIA-NN know the mass using the --mod command.

Best wishes,

Vadim

**Reply by vdemichev** [reply]
That is --mod Deamidated,0.984016

**Reply by vdemichev** [reply]
Also, another general way to deal with this kind of things is to export the library to a .tsv format (using DIA-NN: if it can load a library, it can also export it), and then one can edit it easily in R/Python.

**Reply by brvpuyve** [reply]
Thx Vadim. It works 👍
